geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/pubsub/PubSubExecutor.java

```diff
@@ -33,28 +32,42 @@
   @Override
   public RedisResponse executeCommand(Command command, ExecutionHandlerContext context) {
 
-    byte[] subCommand = command.getProcessedCommand().get(1);
+    List<byte[]> processedCommand = command.getProcessedCommand();
+    String subCommand = Coder.bytesToString(processedCommand.get(1)).toLowerCase();
 
-    if (!Arrays.equals(subCommand, Coder.stringToBytes("channels"))) {
-      return RedisResponse
-          .error(String.format(ERROR_UNKNOWN_PUBSUB_SUBCOMMAND, new String(subCommand)));
-    }
+    switch (subCommand) {
+      case "channels":
+        if (processedCommand.size() > 3) {
+          return RedisResponse.error(String.format(ERROR_UNKNOWN_PUBSUB_SUBCOMMAND, subCommand));
+        }
+
+        List<byte[]> channelsResponse = doChannels(processedCommand, context);
+        return RedisResponse.array(channelsResponse);
 
-    // in a subsequent story, a new parameter requirement class
-    // specific to subCommands might be a better way to do this
-    if (command.getProcessedCommand().size() > 3) {
-      return RedisResponse
-          .error(String.format(ERROR_UNKNOWN_PUBSUB_SUBCOMMAND, new String(subCommand)));
+      case "numsub":
```

Comment:
After rebasing on develop, please add
```
public static final byte[] bNUMSUB = stringToBytes("NUMSUB");
```
to the "PubSubExecutor" section in `StringBytesGlossary` and use the `Coder.equalsIgnoreCaseBytes()` method here.

Reply:
Done